package.json

```diff
@@ -29,6 +29,7 @@
   },
   "homepage": "https://github.com/brentvatne/react-native-scrollable-tab-view#readme",
   "dependencies": {
+    "@react-native-community/viewpager": "^1.1.7",
```

Comment:
```suggestion
    "@react-native-community/viewpager": "^2.0.0",
```

Reply:
I changed version and updated PR. 
Thanks.